client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -108,6 +108,12 @@ class LoggedInForm extends Component {
 			if ( ! isRedirectingToWpAdmin && authorizeSuccess ) {
 				return this.props.goBackToWpAdmin( queryObject.redirect_after_auth );
 			}
+		} else if ( props.isWCS ) {
```

Comment:
Updated this. Didn't put them together at first so it would be clearer the two cases are unrelated, but the comment makes it clear enough and it's not difficult to separate later.